src/impact-dashboard/CurveChartContainer.tsx

```diff
@@ -60,7 +66,7 @@ const CurveChartContainer: React.FC<Props> = ({
     );
   }, [groupStatus, curveData]);
 
-  return modelData.countyLevelDataLoading ? (
+  return !curveDataFiltered ? (
```

Comment:
Much better 🙌 .  Using `countyLevelDataLoading` as a proxy for data loading in the chart was bothering me now that data loading is at the facility level.